src/views/newConnection.tsx

```diff
@@ -0,0 +1,56 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { Page } from './common'
+import { CompanyProfile } from '../models/companyHouseEntity'
+
+
+@singleton()
+export default class newConnectionTemplates {
+  constructor() {}
+
+  public formPage = (company: CompanyProfile, validated: boolean) => {
+    return (
+      <Page title="Veritable - New Connection">
+        <this.CompanyNumberInput validated={false} />
+        <this.CompanyTextBox company={company} />
+      </Page>
+    )
+  }
+
+  public CompanyNumberInput = ({ validated }: { validated: boolean }) => {
+    const attributes: Htmx.Attributes = validated
+      ? {
+          'hx-target': 'closest .button-group',
+          'hx-trigger': 'counter-loaded from:body',
+          'hx-get': '/new-connection/validate-input',
+          'hx-swap': 'outerHTML',
+        }
+      : { 'hx-target': 'closest .button-group', 'hx-post': '/example/button', 'hx-swap': 'outerHTML' }
+
+    return (
+      <div class="input-group">
+        <input validated={validated} {...attributes}>
+          
+        </input>
+      </div>
+    )
+  }
+
+  public CompanyTextBox = ({ company }: { company: CompanyProfile }) => {
+    return (
+      <div class="">
+          <p>{company.company_name}</p>
+          <p>{company.registered_office_address.address_line_1}</p>
+          <p>{company.registered_office_address.address_line_2}</p>
+          <p>{company.registered_office_address.care_of}</p>
+          <p>{company.registered_office_address.country}</p>
+          <p>{company.registered_office_address.locality}</p>
+          <p>{company.registered_office_address.po_box}</p>
+          <p>{company.registered_office_address.postal_code}</p>
+          <p>{company.registered_office_address.premises}</p>
+          <p>{company.registered_office_address.region}</p>
+          <p>{company.company_status}</p>
+      </div>
+    )
+  }
+}
```

Comment:
not sure if UI was part of the story, so in this instance anything will do for as long as we can validate it works. even if it's `JSON.stringify(response)` in plain text also we can verify using the Dev tools "Networking Tab"